tests/parquet_test.py

```diff
@@ -8,16 +8,15 @@
 verbose = True
 
 def compare_values(arr1, arr2):
-    for i in range(len(arr1)):
-        if arr1[i] != arr2[i]:
-            print(arr1[i], 'does not match', arr2[i], 'at index', i)
-            return 1
+    if (arr1 != arr2).all():
```

Comment:
I think this should be `if (arr1 != arr2).any():`. 

As written, arrays will compare equal unless *all* of their values differ.

Reply:
@glitch I believe `arr1` and `arr2` are arkouda arrays. Although you could do `np.array_equal(arr1.to_ndarray(), arr2.to_ndarray())`, I guess I have a personal preference for staying in arkouda to avoid array transfers (even though those are fast now and these arrays are small, so at this point it's mostly a stylistic choice). 

Also, it's been a while since I looked, but I think most of the test suite uses the `(a == b).all()` pattern, so it might be good to use that for consistency.

Reply:
@reuster986 you are correct, `(a == b).all()` is the pattern elsewhere, and it seems that what other tests do most often is just `assertTrue`, rather than do a `compare_values` function like I was doing, so I have switched to that format for the test to be more consistent with other tests.